src/table.rs

```diff
@@ -75,6 +78,33 @@ impl Table {
     }
 }
 
+/// Open an iceberg table by uri
+pub async fn open(uri: impl AsRef<str>) -> Result<Table> {
+    // Todo(xudong): inferring storage types by uri
+    let mut builder = Fs::default();
+    builder.root(uri.as_ref());
+
+    let op = Operator::new(builder)?
+        .layer(LoggingLayer::default())
+        .finish();
+
+    let mut table = Table::new(op);
+    table.load().await?;
+    Ok(table)
+}
+
+impl Display for Table {
```

Comment:
I don't think it's a good idea to implement `Display` for Table. Perhaps we should wait until we find a use case for it before adding it?

Reply:
ok